### PR TITLE
Green light Festival E2E test specs

### PR DESCRIPTION
**Pull request by xmano**
Festival screening invitation spec fully works now.

image

Creating Screening spec - need to debug further failing cases. But it is good to merge.

### CODE REVIEW

File: apps/festival/festival-e2e/src/integration/dashboard/create-screening-2687.spec.ts [hunk 1]
```diff
@@ -42,8 +44,9 @@ describe('User create a screening', () => {
   
     signIn(users[0], true);
 
-    (new FestivalMarketplaceHomePage()).goToDashboard();
-    const marketPage = new FestivalDashboardHomePage();
+    //(new FestivalMarketplaceHomePage()).goToDashboard();
+    //const marketPage = new FestivalDashboardHomePage();
```
Comment (phpgeekfr) [inline apps/festival/festival-e2e/src/integration/dashboard/create-screening-2687.spec.ts:48]: clean useless comments if any

File: libs/e2e/src/lib/fixtures/orgs.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
+﻿import { Organization as OrgType } from '../utils/type';
```
Comment (phpgeekfr) [inline libs/e2e/src/lib/fixtures/orgs.ts:1]: Is this file already used ?

File: libs/e2e/src/lib/fixtures/movies.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
+﻿import { Movie as MovieType } from '../utils/type';
```
Comment (phpgeekfr) [inline libs/e2e/src/lib/fixtures/movies.ts:1]: Is this file already used ?

File: libs/e2e/src/lib/fixtures/movies.ts [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+export class Orgs {
+  constructor() {
+
+  }
+
+  get(query: QueryInferface) : Partial<MovieType>[] {
+    const orgSet: Partial<MovieType>[]  = (query.exist) ? 
+                                      moviesFixture : null;
```
Comment (phpgeekfr) [inline libs/e2e/src/lib/fixtures/movies.ts:17]: only in one line ?

File: libs/e2e/src/lib/fixtures/movies.ts [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+export enum MOVIE {
+  Felicità = '1J5uLFThLziaj2j0xPPP',
+  Hunted = 'KUFRFI3VQ5HLOdymnEX5',
+}
+
+export class Orgs {
+  constructor() {
+
+  }
```
Comment (phpgeekfr) [inline libs/e2e/src/lib/fixtures/movies.ts:13]: useless constructor

**Review by phpgeekfr — APPROVED** [review]

File: apps/festival/festival-e2e/src/integration/dashboard/create-screening-2687.spec.ts (hunk 1 above)
Comment (xmano) [reply apps/festival/festival-e2e/src/integration/dashboard/create-screening-2687.spec.ts:48]: ok

File: libs/e2e/src/lib/fixtures/movies.ts (hunk 3 above)
Comment (xmano) [reply libs/e2e/src/lib/fixtures/movies.ts:1]: corrected

File: libs/e2e/src/lib/fixtures/movies.ts (hunk 5 above)
Comment (xmano) [reply libs/e2e/src/lib/fixtures/movies.ts:13]: corrected

File: libs/e2e/src/lib/fixtures/movies.ts (hunk 4 above)
Comment (xmano) [reply libs/e2e/src/lib/fixtures/movies.ts:17]: ok

**Pull request merged**